### DISCUSSION TITLE
Using pathfindR with different species

### DISCUSSION THREAD

**Discussion by eonurk** [egeulgen/pathfindR#56] [Q&A] [answered]
mm10 would be a good start and support for other gene naming conventions would be awesome.

**Comment by egeulgen** [comment, marked as answer]
hey @eonurk,

pathfindR does support different organisms, see [pathfindR Analysis for non-Homo-sapiens organisms](https://egeulgen.github.io/pathfindR/articles/non_hs_analysis.html). Mmu gene sets (KEGG) and PIN data (STRING) is available within the package.

Using other naming conventions (I'm assuming you mean entrez gene id, ensembl gene id, etc.) is also possible: you only have to supply a custom PIN and custom gene sets with desired ids.

We tried to keep the size of the package to a minimum (as per CRAN policy as well) so only selected human and mouse resources were included as default data. We tried to provide the flexibility for analyzing non-human organisms and other naming conventions by providing the user to supply custom gene sets and PIN data.

Best,
-E
